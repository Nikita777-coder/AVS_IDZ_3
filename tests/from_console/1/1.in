I
0.0001
