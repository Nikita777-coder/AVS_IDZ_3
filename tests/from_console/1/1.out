Изначальный интревал некорректный. Поэтому поиск корня будет вестись в диапозоне [0.000000, 1.000000]
Введите способ приваивания точности поиска корня: Введите точность поиска корня: x = 0.544224
Время выполнения программы без input/output = 0.000013 секунд(ы), а с input/output = 0.000454 секунд(ы)
